src/content/components/footer.md

```diff
@@ -0,0 +1,78 @@
+---
+template: footer
+columns:
+  - title: General
+    links:
+      - label: Dapp
+        url: https://threshold.network/
+        isExternal: true
+      - label: Press
+        url: /press
+        isExternal: false
+      - label: Roadmap
+        url: NEED_URL
+        isExternal: true
+      - label: Blog
+        url: NEED_URL
+        isExternal: true
+  - title: Earn
+    links:
+      - label: Staker
+        url: /earn/staker
+        isExternal: false
+      - label: Liquidity Provider
+        url: "/earn/liquidity-provider "
+        isExternal: false
+      - label: BTC/tBTC User
+        url: /earn/btc
+        isExternal: false
+      - label: Token Holder
+        url: /earn/token-holder
+        isExternal: false
+      - label: Earn Profile Quiz
+        url: /profile-quiz
+        isExternal: false
+  - title: Community
+    links:
+      - label: Governance
+        url: /governance
+        isExternal: false
+      - label: Discord
+        url: NEED_URL
+        isExternal: true
+      - label: Forum
+        url: NEED_URL
+        isExternal: true
+  - title: About
+    links:
+      - label: Contributors
+        url: /contrbutors
+        isExternal: false
+      - label: FAQ
+        url: /faq
+        isExternal: false
+      - label: Audits
+        url: /audits
+        isExternal: false
+social_links:
+  - url: twitter
+    icon:
+      image: /images/twitter.svg
+      alt: twitter
+    label: Twitter
+  - label: Telegram
+    url: telegram
+    icon:
+      image: /images/telegram.svg
+      alt: telegram
+  - label: Discord
+    url: discord
+    icon:
+      image: /images/discord.svg
+      alt: discord
+  - label: Github
+    url: github
+    icon:
+      image: /images/github.svg
+      alt: github
```

Comment:
Maybe we can extract this part to a separate `md` file because we have the same social links at Navbar. But on the other hand we are more flexible in case we want display different social links in footer and navbar. Just flagging.

Reply:
I like the flexibility for now. But I like the thinking... there are definitely other areas where we could dry up our code (like using js instead of yml for the admin file). Probably worth our time updating those before tackling this.